### PR TITLE
Update package.xml

### PR DESCRIPTION
**Pull request by nnmm**
I believe those three lines are required for every message package.

### CODE REVIEW

**Comment by nnmm** [comment]
@audrow Could you review?

**Review by esteve — APPROVED** [review]

File: package.xml [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
   <buildtool_depend>ament_cmake_core</buildtool_depend>
 
+  <build_depend>rosidl_default_generators</build_depend>
```
Comment (esteve) [inline package.xml]: To match other message packages (e.g. https://github.com/ros2/common_interfaces/blob/master/diagnostic_msgs/package.xml), this should be a `<buildtool_depend>`

**Review by esteve — CHANGES_REQUESTED** [review]

**Comment by esteve** [comment]
@audrow any chance this PR could be reviewed soon? If you don't have time, I can review it and merge it. Thanks.

File: package.xml (hunk 1 above)
Comment (nnmm) [reply package.xml]: Done. By the way, I copied this from the ROS 2 tutorial.

**Review by esteve — APPROVED** [review]

**Comment by esteve** [comment]
@nnmm thanks for addressing my feedback. The last commit is missing your signature and it's making the DCO test to fail, can you sign it? Thanks.

**Comment by nnmm** [comment]
Ah yes, my bad. Thanks!

**Comment by esteve** [comment]
@audrow I've spoken with @mxgrey to make sure it's ok if I merge this PR, hope it's ok with you too

**Comment by clalancette** [comment]
So this PR was merged without any CI being run on it, and it is causing us to not be able to build with circular dependencies.  I'm going to open a revert PR, as right now we are totally dead in the water.

**Comment by nnmm** [comment]
@clalancette Sorry for causing so much trouble with this!

**Comment by clalancette** [comment]
> @clalancette Sorry for causing so much trouble with this!

No worries, this should be fixed by #19.

I think it is worthwhile to revisit what you were trying to accomplish here.  You are entirely correct that for most interface packages, the package.xml lines you added are necessary.  We never noticed/needed them here because we don't actually release this package onto the buildfarm, and so nothing complained.

I think the problem with the lines you added is that it introduces a circular dependency between this package and rosidl, which is what colcon was complaining about.  What was the original problem that caused you to examine this and try to add those lines?  Maybe we can find another way to fix it.

**Comment by nnmm** [comment]
@clalancette The purpose was to use this package in https://github.com/ros2-rust/ros2_rust/pull/80 for a demo executable. I was unable to generate Rust code for message definitions in this package and thought the missing dependencies might be the issue. It's probably not (only) the dependencies, but also the way CMakeLists.txt is written. We can always use a custom message package instead of `test_interface_files` however.

**Comment by mxgrey** [comment]
I'm sincerely sorry for the premature go-ahead. The changes seemed perfectly logical and consistent with all other message packages, so I never imagined there would be such an edge case. The green check marks in the GitHub workflow also fooled me into thinking that the CI was okay. I'll make sure I'm familiar with the procedure of running the full CI on a PR in the future :facepalm:

**Comment by esteve** [comment]
@clalancette so sorry about that, I had forgotten about the buildfarm and only looked at the GitHub CI. Thanks for the prompt fix!

**Pull request merged**